sorc/machine-setup.sh

```diff
@@ -28,6 +28,7 @@ if [[ -d /lfs3 ]] ; then
     target=jet
     module purge
     module use /lfs3/projects/hfv3gfs/nwprod/NCEPLIBS/modulefiles
+    module load cmake/3.16.1
```

Comment:
please don't load modules in `machine-setup.sh`
There is a modulefile for each machine (or there was). This belongs there.
This file is to setup a clean module environment.

Reply:
Good point.  Done.